src/app/clusters/window-covering-server/window-covering-server.cpp

```diff
@@ -0,0 +1,685 @@
+    writeAttribute(mEndPoint, ZCL_WC_CURRENT_POSITION_LIFT_PERCENTAGE_ATTRIBUTE_ID, static_cast<uint8_t *> &percent, ZCL_INT8U_ATTRIBUTE_TYPE);
+    writeAttribute(mEndPoint, ZCL_WC_CURRENT_POSITION_LIFT_PERCENT100_THS_ATTRIBUTE_ID, static_cast<uint8_t *> &percent100ths,
+                   ZCL_INT16U_ATTRIBUTE_TYPE);
+    emberAfWindowCoveringClusterPrint("LiftActuator::PositionSet(%u%%)", percent100ths / 100);
+}
+
+uint16_t WindowCover::LiftActuator::PositionGet()
+{
+    uint16_t percent100ths = 0;
+    readAttribute(mEndPoint, ZCL_WC_CURRENT_POSITION_LIFT_PERCENT100_THS_ATTRIBUTE_ID, static_cast<uint8_t *> &percent100ths,
+                  sizeof(uint16_t));
+    return percent100ths;
+}
+
+void WindowCover::LiftActuator::TargetSet(uint16_t percent100ths)
+{
+    emberAfWindowCoveringClusterPrint("LiftActuator::TargetSet(%u%%)", percent100ths / 100);
+    writeAttribute(mEndPoint, ZCL_WC_TARGET_POSITION_LIFT_PERCENT100_THS_ATTRIBUTE_ID, static_cast<uint8_t *>&percent100ths,
+                   ZCL_INT16U_ATTRIBUTE_TYPE);
+}
+
+uint16_t WindowCover::LiftActuator::TargetGet()
+{
+    uint16_t percent100ths = 0;
+    readAttribute(mEndPoint, ZCL_WC_TARGET_POSITION_LIFT_PERCENT100_THS_ATTRIBUTE_ID, static_cast<uint8_t *> &percent100ths, sizeof(uint16_t));
+    return percent100ths;
+}
+
+void WindowCover::LiftActuator::NumberOfActuationsIncrement()
+{
+    uint16_t count = static_cast<uint16_t>(NumberOfActuationsGet() + 1);
+    writeAttribute(mEndPoint, ZCL_WC_NUMBER_OF_ACTUATIONS_LIFT_ATTRIBUTE_ID, static_cast<uint8_t *> &count, ZCL_INT16U_ATTRIBUTE_TYPE);
+}
+
+uint16_t WindowCover::LiftActuator::NumberOfActuationsGet(void)
+{
+    uint16_t count = 0;
+    readAttribute(mEndPoint, ZCL_WC_NUMBER_OF_ACTUATIONS_LIFT_ATTRIBUTE_ID, static_cast<uint8_t *> &count, sizeof(uint16_t));
+    return count;
+}
+
+//------------------------------------------------------------------------------
+// TiltActuator
+//------------------------------------------------------------------------------
+
+void WindowCover::TiltActuator::OpenLimitSet(uint16_t limit)
+{
+    writeAttribute(mEndPoint, ZCL_WC_INSTALLED_OPEN_LIMIT_TILT_ATTRIBUTE_ID, static_cast<uint8_t *> &limit, ZCL_INT16U_ATTRIBUTE_TYPE);
+}
+
+uint16_t WindowCover::TiltActuator::OpenLimitGet(void)
+{
+    uint16_t limit = 0;
+    readAttribute(mEndPoint, ZCL_WC_INSTALLED_OPEN_LIMIT_TILT_ATTRIBUTE_ID, static_cast<uint8_t *> &limit, sizeof(uint16_t));
+    return limit;
+}
+
+void WindowCover::TiltActuator::ClosedLimitSet(uint16_t limit)
+{
+    writeAttribute(mEndPoint, ZCL_WC_INSTALLED_CLOSED_LIMIT_TILT_ATTRIBUTE_ID, static_cast<uint8_t *> &limit, ZCL_INT16U_ATTRIBUTE_TYPE);
+}
+
+uint16_t WindowCover::TiltActuator::ClosedLimitGet(void)
+{
+    uint16_t limit = 0;
+    readAttribute(mEndPoint, ZCL_WC_INSTALLED_CLOSED_LIMIT_TILT_ATTRIBUTE_ID, static_cast<uint8_t *> &limit, sizeof(uint16_t));
```

Comment:
```suggestion
    readAttribute(mEndPoint, ZCL_WC_INSTALLED_CLOSED_LIMIT_TILT_ATTRIBUTE_ID, static_cast<uint8_t *>(&limit), sizeof(uint16_t));
```

Reply:
Replaced with the corresponding Accessors.h function.